src/authenticators/jwt_svid_authenticator/mod.rs

```diff
@@ -0,0 +1,71 @@
+// Copyright 2020 Contributors to the Parsec project.
+// SPDX-License-Identifier: Apache-2.0
+//! JWT SVID authenticator
+
+use super::ApplicationName;
+use super::Authenticate;
+use crate::front::listener::ConnectionMetadata;
+use log::error;
+use parsec_interface::operations::list_authenticators;
+use parsec_interface::requests::request::RequestAuth;
+use parsec_interface::requests::Result;
+use parsec_interface::requests::{AuthType, ResponseStatus};
+use parsec_interface::secrecy::ExposeSecret;
+use spiffe::svid::jwt::Jwt;
+use spiffe::workload::jwt::JWTClient;
+use std::str;
+
+/// JWT SVID authenticator
```

Comment:
Should this go into more detail about what the authenticator does (even if it's fairly straight forward code-wise)?

Reply:
I could! Although similarly than for the Unix Peer Credentials auth, the authenticator is explained in lengths in the book

Reply:
Yeah, I do find this double-documentation that we do around the service a bit weird